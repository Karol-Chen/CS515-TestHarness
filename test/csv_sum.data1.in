name,age,salary
John,25,50000
Jane,30,60000
Bob,22,45000
Alice,28,55000
