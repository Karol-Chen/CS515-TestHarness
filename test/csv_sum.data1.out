,0
age,105
salary,210000
